### PR TITLE
Add swiftui-davinci sample app

### PR DESCRIPTION
**Pull request by vahancouver**
[SDKS-3415](https://pingidentity.atlassian.net/browse/SDKS-3415)
[iOS] [Unified] [DV] - Move DaVinci Sample App into Sample App Repo

### CODE REVIEW

File: iOS/README.md [hunk 1]
```diff
@@ -36,7 +36,10 @@ To try out the Ping iOS SDK please look at one of our samples:
 
 ### Integrate with PingOne:
 
-- Samples coming soon
+-  [**Davinci using SwiftUI - `/ios/swiftui-davinci`**](./swiftui-davinci/)
```
Comment (rodrigoareis) [inline iOS/README.md:39]: Add a README file to the sample app folder. You can follow the structure of the `kotlin-davinci`

File: iOS/swiftui-davinci/Davinci/Davinci/AccessToken.swift [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+//
+//  Copyright (c) 2024 Ping Identity. All rights reserved.
+//
+//  This software may be modified and distributed under the terms
+//  of the MIT license. See the LICENSE file for details.
+//
+
+import SwiftUI
+
+struct AccessTokenView: View {
```
Comment (rodrigoareis) [inline iOS/swiftui-davinci/Davinci/Davinci/AccessToken.swift:13]: Generate docs for the code

**Review by rodrigoareis — CHANGES_REQUESTED** [review]
Overall changes looks good! Left some comments to address

File: iOS/swiftui-davinci/Davinci/Davinci.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -0,0 +1,453 @@
...
+		3A8092F32BE99F0F00AD667F /* InputView.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = InputView.swift; sourceTree = "<group>"; };
+		3A8092F52BE9BB6B00AD667F /* TokenViewModel.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = TokenViewModel.swift; sourceTree = "<group>"; };
+		3A8532322BE2D5D800F8619D /* LoginView.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = LoginView.swift; sourceTree = "<group>"; };
+		3AB1C9F32BD6C6F3003FCE3C /* LoginViewModel.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = LoginViewModel.swift; sourceTree = "<group>"; };
+		3ACD90E02BE586BC00DABCE6 /* DavinciViewModel.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = DavinciViewModel.swift; sourceTree = "<group>"; };
+		3AE945732BF27223007B3381 /* LoginUtility.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = LoginUtility.swift; sourceTree = "<group>"; };
```
Comment (rodrigoareis) [inline iOS/swiftui-davinci/Davinci/Davinci.xcodeproj/project.pbxproj:51]: Remove reference, file was removed

File: iOS/swiftui-davinci/Davinci/Davinci/ContentView.swift [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+        NavigationLink(value: "User") {
+          Text("User Info")
+        }
+        NavigationLink(value: "Logout") {
+          Text("Logout")
+        }
+      }.navigationDestination(for: String.self) { item in
+        switch item {
+        case "Davinci":
+          DavinciView(path: $path)
```
Comment (rodrigoareis) [inline iOS/swiftui-davinci/Davinci/Davinci/ContentView.swift:36]: `DavinciView` is missing. Some others in this switch statement were removed intentionally, so should be removed here.

**Review by rodrigoareis — APPROVED** [review]
Changes looks good to me

**Review by spetrov — APPROVED** [review]
LGTM! 👍🏻

**Pull request merged**